src/creds/cred_manager.rs

```diff
@@ -63,12 +89,12 @@ impl std::fmt::Display for ParseUserRoleError {
 impl std::error::Error for ParseUserRoleError {}
 
 pub struct CredsManager {
-    users: HashMap<String, User>,
-    enable_log: bool,
+    pub users: HashMap<String, User>,
+    pub enable_log: bool,
 }
 
-const ENCRYPT_KEY: &str = "QWERTYUIOPASdfgCBxzjfndeuAQwudhAsd";
-const IV_PATTERN: [u8; 16] = [2, 2, 5, 8, 5, 1, 3, 5, 6, 7, 9, 1, 6, 4, 1, 9];
+pub const ENCRYPT_KEY: &str = "QWERTYUIOPASdfgCBxzjfndeuAQwudhAsd";
+pub const IV_PATTERN: [u8; 16] = [2, 2, 5, 8, 5, 1, 3, 5, 6, 7, 9, 1, 6, 4, 1, 9];
```

Comment:
we should'nt hard code `ENCRYPT_KEY` and `IV_PATTERN`. we should have these in the configuration file. It should be optional.
Some random `ENCRYPT_KEY` and `IV_PATTERN` should be created if not provided.

Reply:
no it must do by product owner not in configuration cause when we want to sell this product to BS we need handle by our selves, not client .in open source version they go to code and change it

Reply:
I made issue #45 for this